dbt/clients/jinja.py

```diff
@@ -144,19 +139,15 @@ def get_template(string, ctx, node=None, capture_macros=False,
         if capture_macros:
             args['undefined'] = create_macro_capture_env(node)
 
-        if validate_macro:
-            args['extensions'].append(create_macro_validation_extension(node))
-
         args['extensions'].append(MaterializationExtension)
-        args['extensions'].append(
-            create_statement_extension(node, ctx, execute_statements))
 
-        env = jinja2.sandbox.SandboxedEnvironment(**args)
+        env = MacroFuzzEnvironment(**args)
 
         return env.from_string(dbt.compat.to_string(string), globals=ctx)
 
     except (jinja2.exceptions.TemplateSyntaxError,
             jinja2.exceptions.UndefinedError) as e:
+        e.translated = False
```

Comment:
it includes the affected line number and code from the source template in the exception msg